js/pulse.js

```diff
@@ -359,10 +359,14 @@
 
                 if(!this._pulseHost) {
                     log('No Pulse hostname found in plugin parameters or media metadata; will not attempt to show Pulse ads');
+                    this.ready = true;
+                    amc.onAdManagerReady();
                     return;
                 }
 
                 this.ready = true;
+                amc.onAdManagerReady();
```

Comment:
Is it safe to call onAdManagerReady here? When we call this API, the AMC checks to see if all ad plugins are ready. If so, it unblocks playback ready. At this point, playback can occur when the user clicks the play icon or automatically if autoplay is on.